js/mercado.js

```diff
@@ -42,6 +71,7 @@ module.exports = class mercado extends Exchange {
                         '{coin}/trades/{from}/',
                         '{coin}/trades/{from}/{to}',
                         '{coin}/day-summary/{year}/{month}/{day}/',
+
```

Comment:
Please delete this empty line from inside the `describe()` method, because it is violating the CONTRIBUTING guidelines from here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

> don't put empty lines inside your methods

Otherwise it breaks down the transpiler.